src/lib/utils/notifications/index.ts

```diff
@@ -1,15 +1,15 @@
-import sendPushNotifications from "$lib/utils/notifications/push";
+// import sendPushNotifications from "$lib/utils/notifications/push";
 import {
+  NotificationSettingType,
   NotificationType,
   SUBSCRIPTION_SETTINGS_MAP,
-  type NotificationSettingType,
 } from "$lib/utils/notifications/types";
 import type { Author, PrismaClient } from "@prisma/client";
 import { error } from "@sveltejs/kit";
 
 const DUPLICATE_ALLOWED_TYPES = [
   NotificationType.CREATE_MANDATE,
-  NotificationType.ARTICLE_UPDATE,
+  NotificationType.ARTICLE_REQUEST_UPDATE,
```

Comment:
I renamed this to be more clear. `ARTICLE_REQUEST_UPDATE` (as it's now called) is meant to be a notification you get when an update has been made to your active article request (not yet built in this version). Such as if it has been approved or rejected.